src/game/shared/tf/passtime_convars.cpp

```diff
@@ -12,81 +12,82 @@
 
 #define PASSTIME_CONVAR(NAME, STR, DESC) ConVar NAME(#NAME, #STR, FLAGS_DEFAULT, DESC)
 
-PASSTIME_CONVAR( tf_passtime_scores_per_round, 5, "Number of scores it takes to win a round. Similar to tf_flag_caps_per_round." );
```

Comment:
One day I feel we ought to clang-format the world to avoid these kinds of diffs from sneaking in. (no action needed)